test/jobs/fetch_faculty_data_job_test.rb

```diff
@@ -71,7 +71,6 @@ def expected_building_names
         "Crean Hall",
         "DeMille Hall",
         "Digital Media Arts Center",
-        "Dodge College",
```

Comment:
Yea, VCR allows us to save a one-time request response I'm pretty sure. That way we can just have the same static data every time while keeping it accurate and closely resembling the production data we'll be receiving.